stdlib/public/Platform/tgmath.swift.gyb

```diff
@@ -14,8 +14,8 @@ import SwiftShims
 
 // Generic functions implementable directly on FloatingPoint.
 @_transparent
-public func fabs<T: FloatingPoint>(_ x: T) -> T
-  where T.Magnitude == T {
+@available(swift, deprecated: 4.2, renamed: "abs")
```

Comment:
Q: why deprecate this specifically?

Reply:
For the same reason that we deprecate all the `tgmath` overlays whose functionality duplicates APIs available on `[Binary]FloatingPoint`. We don't need to carry two names for the same function into the future, and `abs` is the better name for this. `fabs` is a curious artifact of the fact that C doesn't have overloaded functions.